modules/special-rules/pages/pulsar-x.adoc

```diff
@@ -0,0 +1,19 @@
+= Pulsar (X)
+
+This weapon is unique to the Eldar race, and it demonstrates their mastery of laser weaponry.
+
+The weapon shoots in rapid but powerful bursts, guided by advanced targeting technologies.
+As a result, the weapon can destroy multiple targets in a single salvo.
+
+---
+
+When you shoot with a {pulsar-x}:
```

Comment:
Hmmm... we previously figured out a consistent way to construct the lead-in sentence for super-heavy weapons, in part to overcome difficulties in relation to 'weapon systems' — but we broke it for _Pulsar_ because of the reference to 'shots'. If instead we apply the same construction here, we're more consistent and we solves the 'prose/not prose hybrid usage' problem we've been discussing at https://github.com/GameBrains/er-core/pull/43/files#r1177022647.

```suggestion
When you make a {pulsar-x} shot:
```

However, this makes the mention of 'shots' a little odd in the bullet that follows, so I think we'll need to tweak that a little more to work with this suggestion.